.github/workflows/build_and_test.yml

```diff
@@ -339,6 +339,8 @@ jobs:
             pyspark-pandas-slow
           - >-
             pyspark-connect, pyspark-errors
+          - >-
+            pyspark-pandas-connect
```

Comment:
I noticed that there are two test modules for pandas: 

- pyspark-pandas
- pyspark-pandas-slow

shall we create two new ones for them? I guess `pyspark-pandas-connect` will still be slow after we reuse and reenable most pandas UTs.

Reply:
Yes, but let's see if the current way is enough or not. I assume many of pandas-connect tests are still skipped, then this is enough for now.
I'm worrying that too many parallel tasks could hit some limitation of GHA.

Reply:
ok, sounds good.